kartothek/io_components/cube/write.py

```diff
@@ -201,18 +233,15 @@ def _check_user_df(ktk_cube_dataset_id, df, cube, existing_payload, partition_on
             )
         )
 
-    for cols, what in (
-        (partition_on, "partition"),
-        (dcols_present, "dimension"),
-        (icols_present, "index"),
-    ):
-        for col in sorted(cols):
-            if df[col].isnull().any():
-                raise ValueError(
-                    'Found NULL-values in {what} column "{col}" of dataset "{ktk_cube_dataset_id}"'.format(
-                        col=col, ktk_cube_dataset_id=ktk_cube_dataset_id, what=what
-                    )
-                )
+    # Factor this check out. All others can be performed on the dask.DataFrame.
+    # This one can only be executed on a pandas DataFame
+    if isinstance(df, pd.DataFrame):
```

Comment:
So this check is just silently skipped in dask dataframes?

Reply:
see other comment